src/screens/Map/components/MapDisabledForScreenReader.tsx

```diff
@@ -0,0 +1,37 @@
+import ThemeText from '@atb/components/text';
+import {StyleSheet} from '@atb/theme';
+import {ThemedMapImage} from '@atb/theme/ThemedAssets';
+import {MapTexts, useTranslation} from '@atb/translations';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+export function MapDisabledForScreenReader() {
+  const styles = useStyles();
+  const {t} = useTranslation();
+  return (
+    <SafeAreaView style={styles.container}>
+      <ThemedMapImage />
+      <ThemeText type="body__primary--bold" style={styles.header}>
+        {t(MapTexts.disabledForScreenReader.title)}
+      </ThemeText>
+      <ThemeText style={styles.description}>
+        {t(MapTexts.disabledForScreenReader.description)}
+      </ThemeText>
```

Comment:
I'll suggest that we add an accessible `View` around all this, with an a11y label reading both the title and description.

Reply:
Updated here 1c95aae72a955775f427cf5e021c2fd92c5dd842.

Reply:
Nice. Tested on Android?

Reply:
I don't have a device to test accessibility on android. Is there any known differences between iOS and android here that I should be aware of?

Reply:
Yes 😅 Should try to borrow one from Tor.

Reply:
Works well on Android ✅